source/locale/pt_BR/LC_MESSAGES/concepts.po

```diff
@@ -124,11 +124,83 @@ msgstr ""
 "pode encontrar um tutorial de como autenticar um usuário e obter um token"
 " de acesso na `auth documentation`_."
 
-#: ../../source/concepts.rst:56
+#: ../../source/concepts.rst:51
+#, fuzzy
```

Comment:
When has `#, fuzzy`, so probably has a problem.
You can fix and remove the line with the fuzzy

Reply:
You're right, `gettext` tried to infer a translation automatically ...